spec/cypress/integration/admin/LeadProviderUserManagement.feature

```diff
@@ -0,0 +1,73 @@
+Feature: Admin user modifying lead provider users
+  Admin users should be able to create and delete lead provider users
+
+  Background:
+    Given I am logged in as an "admin"
+    And scenario "admin/suppliers" has been run
+    And I am on "lead provider users index" page
+
+  Scenario: Creating a new lead provider user
+    When I click on "create supplier user button"
+    Then I should be on "new lead provider user" page
+    And the page should be accessible
+
+    When I type "Lead" into "supplier name input"
+    And I click on "li" containing "Lead Provider 1"
```

Comment:
huh, is this because we are selecting from dropdown?

Can we write it as `I select "Lead Provider 1" from dropdown "xxx"`?

Reply:
oh yea i meant to come back to this